ios/startsession.go

```diff
@@ -52,6 +54,9 @@ func (lockDownConn *LockDownConnection) StartSession(pairRecord PairRecord) (Sta
 		return StartSessionResponse{}, err
 	}
 	response := startSessionResponsefromBytes(resp)
+	if response.Error != "" {
+		return StartSessionResponse{}, errors.New(response.Error)
```

Comment:
lets add a bit more context to the error
```suggestion
		return StartSessionResponse{}, fmt.Errorf("failed to start new lockdown session: %s", response.Error)
```